content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/clientlibs/site/js/image.js

```diff
@@ -372,7 +444,7 @@
     if (document.readyState !== "loading") {
         onDocumentReady();
     } else {
-        document.addEventListener("DOMContentLoaded", onDocumentReady);
+        document.addEventListener("DOMContentLoaded", onDocumentReady());
```

Comment:
This line should be reverted - guessing you started from an old version of the file.  This was corrected in February 2019.